tools/mo/CMakeLists.txt

```diff
@@ -22,6 +22,12 @@ configure_file("${CMAKE_CURRENT_SOURCE_DIR}/automation/version.txt.in"
 
 if(ENABLE_TESTS)
     install(FILES requirements_dev.txt
+                  requirements_caffe.txt
```

Comment:
the main problem here is that these files refer to `-c ../contrainsts.txt` files, which is relative path. Once we install it into test package, this relative path is not valid anymore 😢

Reply:
ok, then you need to install this dependency file (`constants.txt`) as well

Reply:
yes , it is already there )